chia/rpc/wallet_rpc_api.py

```diff
@@ -1066,13 +1067,62 @@ async def cancel_offer(self, request: Dict) -> EndpointResult:
         secure = request["secure"]
         trade_id = bytes32.from_hexstr(request["trade_id"])
         fee: uint64 = uint64(request.get("fee", 0))
-
+        txs = None
         async with self.service.wallet_state_manager.lock:
             if secure:
-                await wsm.trade_manager.cancel_pending_offer_safely(bytes32(trade_id), fee=fee)
+              txs = await wsm.trade_manager.cancel_pending_offer_safely(bytes32(trade_id), fee=fee)
             else:
                 await wsm.trade_manager.cancel_pending_offer(bytes32(trade_id))
-        return {}
+        return {"success": True, "TXs": txs}
```

Comment:
Not a necessary change, but we don't need `"success": True`, the RPC API class takes care of that automatically for us.